src/wp-includes/rest-api/endpoints/class-wp-rest-font-collections-controller.php

```diff
@@ -0,0 +1,333 @@
+										'description' => __( 'Font family string.' ),
+										'type'        => 'string',
+									),
+									'slug'       => array(
+										'description' => __( 'Slug of the font family.' ),
+										'type'        => 'string',
+									),
+									'category'   => array(
+										'description' => __( 'Category of the font family.' ),
+										'type'        => 'string',
+									),
+									'font_face'   => array(
+										'description' => __( 'Font face details.' ),
+										'type'        => 'array',
+										'items'       => array(
+											'type'       => 'object',
+											'properties' => array(
+												'download_from_url' => array(
+													'description' => __( 'URL to download the font.' ),
+													'type' => 'string',
+												),
+												'font_weight' => array(
+													'description' => __( 'Font weight.' ),
+													'type' => 'string',
+												),
+												'fonts_style' => array(
+													'description' => __( 'Font style.' ),
+													'type' => 'string',
+												),
+												'font_family' => array(
+													'description' => __( 'Font family string.' ),
+													'type' => 'string',
+												),
+												'preview' => array(
+													'description' => __( 'URL for font preview.' ),
+													'type' => 'string',
+												),
+											),
+										),
+									),
+									'preview'    => array(
+										'description' => __( 'URL for font family preview.' ),
+										'type'        => 'string',
+									),
+								),
+							),
+						),
+					),
+				),
+			),
+		);
+
+		$this->schema = $schema;
+		return $this->add_additional_fields_schema( $this->schema );
+	}
+
+	/**
+	 * Convert string from camelCase to snake_case.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param string $input String to convert.
+	 * @return string Converted string.
+	 */
+	private function camel_to_snake( $input ) {
+		$output = _wp_to_kebab_case( $input );
+		return str_replace( '-', '_', $output );
+	}
+	
+	/**
+	 * Convert array keys from camelCase to snake_case.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param array $array Array to convert.
+	 * @return array Converted array.
+	 */
+	private function array_keys_to_snake_case( $array ) {
+		$new_array = [];
+		foreach ( $array as $key => $value ) {
+			$snake_key = $this->camel_to_snake( $key );
+			// If the value is an array, recurse.
+			if ( is_array( $value ) ) {
+				$value = $this->array_keys_to_snake_case( $value );
+			}
+			// Add to the new array
+			$new_array[ $snake_key ] = $value;
+		}
+		return $new_array;
+	}
+
+	/**
+	 * Prepares a single font collection output for response.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param array $item Font collection data.
+	 * @param WP_REST_Request $request Request object.
+	 * @return array Font collection data.
+	 */
+	public function prepare_item_for_response( $item, $request ) {
+		return $this->array_keys_to_snake_case( $item );
```

Comment:
What about when you pass the ?_fields parameter ? You need code like this 

https://github.com/WordPress/wordpress-develop/blob/23c811e59fe3a634800e3672df7dc8d45556ae6a/src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php#L1757-L1764

Reply:
This is working out of the box. Example: 

https://github.com/WordPress/wordpress-develop/assets/1310626/a5db2e74-96a7-41ae-bbed-2a461b15e73f

Reply:
@TimothyBJacobs Does it filter the filters by default?

Reply:
I see this is included at

https://github.com/WordPress/gutenberg/blob/066f8777b113ca8a71c79deb052d1deb23fe9860/lib/compat/wordpress-6.5/fonts/class-wp-rest-font-collections-controller.php#L166-L171